components/table/table-wrapper.js

```diff
@@ -62,6 +63,16 @@ export const tableStyles = css`
 	th.d2l-table-header-col-sortable {
 		padding: 0;
 	}
+	.d2l-checkbox,
+	d2l-input-checkbox,
+	d2l-selection-select-all,
+	input[type="checkbox"] {
```

Comment:
Looking at the `input[type="checkbox"]` in the vdiffs, it's already quite short so I think we can leave it out at this point.